extensions/sql-migration/src/wizard/loginMigrationStatusPage.ts

```diff
@@ -0,0 +1,377 @@
+				forceFitColumns: azdata.ColumnSizingMode.ForceFit,
+				columns: [
+					{
+						name: constants.SOURCE_LOGIN,
+						value: 'sourceLogin',
+						type: azdata.ColumnType.text,
+						width: 360,
+						cssClass: cssClass,
+						headerCssClass: cssClass,
+					},
+					{
+						name: constants.LOGIN_TYPE,
+						value: 'loginType',
+						type: azdata.ColumnType.text,
+						width: 80,
+						cssClass: cssClass,
+						headerCssClass: cssClass,
+					},
+					{
+						name: constants.DEFAULT_DATABASE,
+						value: 'defaultDatabase',
+						type: azdata.ColumnType.text,
+						width: 80,
+						cssClass: cssClass,
+						headerCssClass: cssClass,
+					},
+					<azdata.HyperlinkColumn>{
+						name: constants.LOGIN_MIGRATION_STATUS_COLUMN,
+						value: 'migrationStatus',
+						width: 200,
+						type: azdata.ColumnType.hyperlink,
+						icon: IconPathHelper.inProgressMigration,
+						showText: true,
+						cssClass: cssClass,
+						headerCssClass: cssClass,
+					},
+				]
+			}).component();
+
+		this._disposables.push(
+			this._migratingLoginsTable.onCellAction!(async (rowState: azdata.ICellActionEventArgs) => {
+				const buttonState = <azdata.ICellActionEventArgs>rowState;
+				switch (buttonState?.column) {
+					case 3:
+						const loginName = this._migratingLoginsTable!.data[rowState.row][0];
+						const status = this._migratingLoginsTable!.data[rowState.row][3].title;
+						const statusMessage = constants.DATABASE_MIGRATION_STATUS_LABEL(status);
+						var errors = "";
+
+						if (this.migrationStateModel._loginMigrationsResult?.exceptionMap) {
+							const exception_key = Object.keys(this.migrationStateModel._loginMigrationsResult.exceptionMap).find(key => key.toLocaleLowerCase() === loginName.toLocaleLowerCase());
+							if (exception_key) {
+								errors = this.migrationStateModel._loginMigrationsResult.exceptionMap[exception_key][0].Message;
+							}
+						}
+
+						this.showDialogMessage(
+							constants.DATABASE_MIGRATION_STATUS_TITLE,
+							statusMessage,
+							errors);
+						break;
+				}
+			}));
+
+		// load unfiltered table list
+		await this._filterTableList('');
+
+		const flex = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			height: '100%',
+		}).withProps({
+			CSSStyles: {
+				'margin': '0px 28px 0px 28px'
+			}
+		}).component();
+		flex.addItem(this._progressContainer, { flex: '0 0 auto' });
+		flex.addItem(this.createSearchComponent(), { flex: '0 0 auto' });
+		flex.addItem(this._loginCount, { flex: '0 0 auto' });
+		flex.addItem(this._migratingLoginsTable);
+		return flex;
+	}
+
+	private async _loadMigratingLoginsList(stateMachine: MigrationStateModel): Promise<void> {
+		const loginList = stateMachine._loginsForMigration || [];
+		loginList.sort((a, b) => a.loginName.localeCompare(b.loginName));
+
+		this._loginsTableValues = loginList.map(login => {
+			const loginName = login.loginName;
+
+			var status = "InProgress";
+			var title = "In progress";
```

Comment:
can 
* status be set using constant names like LoginMigrationState.InProgress
* title with a resource string